runner/tests/tests.rs

```diff
@@ -174,6 +174,14 @@ fn test_shor() -> Result<(), String> {
     run_bitcode(bitcode, None, 1)
 }
 
+// This test verifies the behavior of measurement in the Pauli-Y basis using basis changes, assertions,
+// and repeated measurements.
+#[test]
+fn test_measure_pauli_y() -> Result<(), String> {
```

Comment:
Does it make sense for this test to be in bitcode or rust making the rt calls?

Reply:
I'm not a fan of bitcode tests (unless specifically testing bitcode reading/writing functionality) because they're impossible to read while reviewing...

Reply:
@samarsha That was my thought as well. This isn't really testing any bitcode interaction, and being in bitcode, what is being tested is opaque.